docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -155,13 +155,7 @@ Install the vSphere CPI and the CSI driver using the zone and region entries.
 
    Make sure `external-provisioner` is deployed with the arguments `--feature-gates=Topology=true`.
 
-   In the credential secret file, add entries for region and zone.
-
-   ```bash
-   [Labels]
-   region = k8s-region
-   zone = k8s-zone
-   ```
+   For topology aware clusters, the CSI Node DaemonSet needs to be started with the `TOPOLOGY_AWARE` environment variable. This makes use of the node labels for topology placement.
```

Comment:
I think doc should be updated when change is available in release, otherwise, `v2.0.0` users will get confused.

Reply:
Thats allright. I'll update the PR and remove the changes to the docs :+1: